macOS arm64: sage.libs.braiding build failure

```
  building 'sage.libs.braiding' extension
  creating build/temp.macosx-11.0-arm64-cpython-39/sage/libs
  g++ -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -g -arch arm64 -I/private/var/folders/0w/4z5l9vds32nbkz7l22n8j6s80000gn/T/build-env-o6r4msm7/lib/python3.9/site-packages/cysignals -I/private/var/folders/0w/4z5l9vds32nbkz7l22n8j6s80000gn/T/build-env-o6r4msm7/lib/python3.9/site-packages -I/Library/Frameworks/Python.framework/Versions/3.9/include/python3.9 -I/private/var/folders/0w/4z5l9vds32nbkz7l22n8j6s80000gn/T/build-env-o6r4msm7/include -I/Library/Frameworks/Python.framework/Versions/3.9/include/python3.9 -c sage/libs/braiding.cpp -o build/temp.macosx-11.0-arm64-cpython-39/sage/libs/braiding.o
  In file included from sage/libs/braiding.cpp:2195:
  /Users/runner/work/passagemath/passagemath/prefix/include/braiding.h:833:22: error: a space is required between consecutive right angle brackets (use '> >')
  list<list<list<sint16>>> SendToSSS(int n, list<sint16> word);
                       ^~
                       > > 
```
https://github.com/passagemath/passagemath/actions/runs/12230785773/job/34151257172#step:34:298

--- a/src/sage/libs/braiding.pyx
+++ b/src/sage/libs/braiding.pyx
@@ -1,6 +1,7 @@
 # sage_setup: distribution = sagemath-libbraiding
 # distutils: libraries = braiding
 # distutils: language = c++
+# distutils: extra_compile_args = -std=c++11
 r"""
 Cython wrapper for the libbraiding library.
 

